eng/pipelines/mirror-container-images.yml

```diff
@@ -0,0 +1,39 @@
+trigger: none
+pr: none
+
+parameters:
+  - name: Images
+    type: object
+    default:
+      - source: ghcr.io/chaos-mesh/chaos-daemon:v2.1.4
+        mirror: azsdkengsys.azurecr.io/mirror/chaos-mesh/chaos-daemon:v2.1.4
+      - source: ghcr.io/chaos-mesh/chaos-mesh:v2.1.4
+        mirror: azsdkengsys.azurecr.io/mirror/chaos-mesh/chaos-mesh:v2.1.4
+      - source: ghcr.io/chaos-mesh/chaos-dashboard:v2.1.4
+        mirror: azsdkengsys.azurecr.io/mirror/chaos-mesh/chaos-dashboard:v2.1.4
+      - source: ubuntu/squid
```

Comment:
Used by c++, an alternative is we could put one of these pipelines in each language repository.